### ISSUE TITLE
Multiply repot file merging with multimodule project reports aggregation

### ISSUE THREAD

**Issue by KetothXupack** [jacoco/jacoco#69]
Hi! Our team using 3 types of jacoco reports in our projects
- integration tests coverage
- unit tests coverage
- it+unit coverage

We used jacoco ant tasks for merging/aggregating job for a while but ended up with this patch: https://gist.github.com/4491163

Configuration example 
```xml
<configuration>

  <!-- data files to merge -->
  <dataFiles>
    <dataFile>${jacoco.exec.it}</dataFile>
    <dataFile>${jacoco.exec.unit}</dataFile>
  </dataFiles>

  <!-- fail on missing data file from list -->
  <strict>false</strict>

  <!-- skip report generating for each module in 
       multimodule environment -->
  <skipModule>true</skipModule>

  <!-- ...and aggregate all module reports in superpom porject -->
  <aggregate>true</aggregate>
</configuration>
```

This patch does fallback to old behavior till no data passed to `<dataFiles/>`.

Anyway patch is rough enough and there is no tests yet... So any modifications are welcome!

Also it would be nice if we could combine this patch with patch from https://github.com/jacoco/jacoco/issues/12.

**Comment by SmiddyPence** [comment]
+1 To Support this.  Currently having to work around.

**Comment by marchof** [comment]
Fixed with #388.
